### PR TITLE
PAYARA-4099 Document Temp Nodes

### PR DESCRIPTION
**Pull request by Pandrex247**
See parent PR: https://github.com/payara/Payara/pull/4260

### CODE REVIEW

File: documentation/payara-server/docker/docker-instances.adoc [hunk 1]
```diff
@@ -106,14 +106,21 @@ instance using the port listed in its config.
...
 [[creating-instances-from-docker]]
 == Creating Instances from Docker
-For Payara Server 5.193, the default Docker Image now supports creating instances from Docker itself, making use of the
-link:documentation/payara-server/asadmin-commands/auto-naming.adoc[auto-naming] feature to resolve any conflicts. This
-opens up the possibility for limited autoscaling to be achieved; instances are still tied to nodes however, and so any
-scaling must be done on the same machine. When instances are created this way, unless specified the container name will
-not match that of the instance.
+Payara Server supports creating instances from Docker itself, making use of the
+link:documentation/payara-server/asadmin-commands/auto-naming.adoc[auto-naming] feature to resolve any conflicts. When
+instances are created this way, unless specified the container name will not match that of the instance.
```
[the remark this answers is not in the document]
Comment (OndroMih) [reply documentation/payara-server/docker/docker-instances.adoc]: I still don't understand what this sentence is trying to say. What's the gist of its message? That the name of the created Docker container will be different than the generated name of the instance? Is that information useful? I don't see that anything would imply that the container name would be equal to the instance name, that's why it confuses me. If it's just a side note for the reader, then we can turn it into a NOTE with https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#admon-bl

File: documentation/payara-server/docker/docker-instances.adoc [hunk 2]
```diff
@@ -101,19 +101,63 @@ See below for some examples:
...
+Payara Server supports creating instances from Docker itself, making use of the
+link:documentation/payara-server/asadmin-commands/auto-naming.adoc[auto-naming] feature to resolve any conflicts. When
+instances are created this way, unless specified in the Docker container configuration itself, the container name will
 not match that of the instance.
 
-IMPORTANT: As alluded to in the previous paragraph, a Docker Node must be registered to the DAS to create instances
-from Docker. The instances must also be housed on the machine that maps to this Docker Node.
+If no or incorrect node details are given, Payara Server will create a temporary node with an autogenerated name.
+Temporary nodes only exist as long as they have instances registered to them, and are cleaned up once the last instance
+registered to the DAS is stopped or deleted (via asadmin or admin console). Temporary nodes are also cleaned up on
```
Comment (dmatej) [inline documentation/payara-server/docker/docker-instances.adoc]: They are cleaned up only when stopped via DAS. When stopped directly via docker, DAS will only mark them as stopped but does not delete them.

**Comment by Pandrex247** [comment]
You should have edit permissions

**Comment by dmatej** [comment]
I'm still working on that - I had to test it again and again and make notes :)

**Comment by Pandrex247** [comment]
Carried on here: https://github.com/payara/Payara-Server-Documentation/pull/651#pullrequestreview-311167287

**Pull request closed without merging**